src/scene/vcModel.cpp

```diff
@@ -488,7 +488,7 @@ void vcModel::HandleContextMenu(vcState *pProgramState)
   ImGui::Separator();
 
 #if VC_HASCONVERT
-  if (((m_pPreferredProjection == nullptr && pProgramState->gis.SRID == 0) || (m_pPreferredProjection != nullptr && m_pPreferredProjection->srid == pProgramState->gis.SRID)) && (m_defaultMatrix == m_sceneMatrix))
+  if ((m_pPreferredProjection == nullptr && pProgramState->gis.SRID == 0) || (m_pPreferredProjection != nullptr && m_pPreferredProjection->srid == pProgramState->gis.SRID))
```

Comment:
That check is required, otherwise the model may be exported with a matrix that isn't the one used when rendering in the scene. Is this just wrong by a tiny amount and we might need to do a "equal within tolerance" check?

Reply:
Yes, t.x is slightly different. I can add a special check here.

Reply:
Restored the matrix equal check.